cdk/lib/frontend.ts

```diff
@@ -18,5 +34,360 @@ export class Frontend extends GuStack {
+        metricName: "HealthyHostCount",
+        namespace: "AWS/ApplicationELB",
+        dimensionsMap: {
+          LoadBalancer: ec2App.loadBalancer.loadBalancerFullName,
+          TargetGroup: ec2App.targetGroup.targetGroupFullName,
+        },
+        statistic: "Average",
+        period: Duration.seconds(60),
+      }),
+      snsTopicName: "reader-revenue-dev",
+    });
+
+    new GuAlarm(this, "ReducedHealthyInstancesAlarm", {
+      app,
+      alarmName: alarmName(
+        "reduced number healthy instances for support-frontend"
+      ),
+      alarmDescription: alarmDescription(
+        "Imminent issue cannot sell any subscriptions or contributions products"
+      ),
+      actionsEnabled: alarmEnabledInProd,
+      threshold: 2,
+      evaluationPeriods: 2,
+      comparisonOperator: ComparisonOperator.LESS_THAN_OR_EQUAL_TO_THRESHOLD,
+      metric: new Metric({
+        metricName: "HealthyHostCount",
+        namespace: "AWS/ApplicationELB",
+        dimensionsMap: {
+          LoadBalancer: ec2App.loadBalancer.loadBalancerFullName,
+          TargetGroup: ec2App.targetGroup.targetGroupFullName,
+        },
+        statistic: "Average",
+        period: Duration.seconds(300),
+      }),
+      snsTopicName: "reader-revenue-dev",
+    });
+
+    new GuAlarm(this, "High5XXRateAlarm", {
+      app,
+      alarmName: alarmName(
+        "support-frontend instances are returning 5XX errors"
+      ),
+      alarmDescription: alarmDescription(
+        "Some or all actions on support website are failing"
+      ),
+      actionsEnabled: alarmEnabledInProd,
+      threshold: 3,
+      evaluationPeriods: 2,
+      comparisonOperator: ComparisonOperator.GREATER_THAN_OR_EQUAL_TO_THRESHOLD,
+      metric: new Metric({
+        metricName: "HTTPCode_Target_5XX_Count",
+        namespace: "AWS/ApplicationELB",
+        dimensionsMap: {
+          LoadBalancer: ec2App.loadBalancer.loadBalancerFullName,
+          TargetGroup: ec2App.targetGroup.targetGroupFullName,
+        },
+        statistic: "Sum",
+        period: Duration.seconds(60),
+      }),
+      snsTopicName: "reader-revenue-dev",
+    });
+
+    new GuAlarm(this, "HighELB5XXRateAlarm", {
+      app,
+      alarmName: alarmName("support-frontend ELB is returning 5XX errors"),
+      alarmDescription: alarmDescription(
+        "Some or all actions on support website are failing"
+      ),
+      actionsEnabled: alarmEnabledInProd,
+      threshold: 3,
+      evaluationPeriods: 2,
+      comparisonOperator: ComparisonOperator.GREATER_THAN_OR_EQUAL_TO_THRESHOLD,
+      metric: new Metric({
+        metricName: "HTTPCode_ELB_5XX_Count",
+        namespace: "AWS/ApplicationELB",
+        dimensionsMap: {
+          LoadBalancer: ec2App.loadBalancer.loadBalancerFullName,
+        },
+        statistic: "Sum",
+        period: Duration.seconds(60),
+      }),
+      snsTopicName: "reader-revenue-dev",
+    });
+
+    new GuAlarm(this, "LatencyNotificationAlarm", {
```

Comment:
Interesting to see that you are alerting on latency; I wonder if we should provide this as standard @nicl @akash1810 @michaelwmcnamara?

Reply:
Yeah, this makes sense as latency is a form of availability. Frontend do something similar:

https://github.com/guardian/platform/blob/main/provisioning/cloudformation/frontend-service.yaml#L401